chore: remove compression

loks0n opened the pull request:
Remove the storage features from `utopia-php/storage`, this functionality now lives in https://github.com/utopia-php/compression, to enable libraries, mainly `utopia-php/http` to depend on compression without storage.

lohanidamodar commented:
@loks0n can you add description, any more details on why this decision was made?

loks0n commented:
> @loks0n can you add description, any more details on why this decision was made?

We've extracted this to https://github.com/utopia-php/compression, so we can use the functionality in utopia-php/http without depending on storage. The new compression library has also added some additional features and some more compression libraries.
